web/wp-content/themes/bumblebee/single.php

```diff
@@ -130,6 +130,13 @@ function () {
 					</aside>
 				</div>
 			</div>
+			<div class="pure-u-md-3-24 pure-u-lg-3-24 pure-u-xl-3-24 hide-on-mobile"></div>
```

Comment:
What is this empty div for?

Reply:
It's the (empty) left sidebar, I was following on from how the template was coded